build/scripts/gofmt.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh	
+
+out=$(find . -name '*.go' -type f | xargs gofmt -l -s | grep -v vendor)
```

Comment:
Should we use `egrep -v '^vendor/'` just to be extra cautious?

I think the `grep` and `xargs` are out of order, too.

`out=$(find . -name '*.go' -type f | egrep -v '^vendor/' | xargs gofmt -l -s)`